action.yml

```
@@ -14,9 +17,9 @@ inputs:
   client-secret:
     description: "Your DevCycle API client secret (Recommended)"
     required: false
-  github-token:
-    description: "The GitHub token for your repo"
-    required: true
+  only-comment-on-change:
+    description: "If true, comments will only be added to PRs when changes to DevCycle variables are detected. Defaults to false."
+    required: false
```

@shoodey could you also add a `default: false` here to ensure that the `getBooleanInput` is properly set. This should resolve the failed check: https://github.com/DevCycleHQ/feature-flag-pr-insights-action/actions/runs/19170226865/job/54810109911?pr=117

```suggestion
    required: false
    default: false
```